buildconfig/CMake/CppCheck_Suppressions.txt.in

```diff
@@ -319,7 +319,7 @@ constVariablePointer:${CMAKE_SOURCE_DIR}/Framework/API/src/CompositeFunction.cpp
 unusedScopedObject:${CMAKE_SOURCE_DIR}/Framework/Kernel/inc/MantidKernel/NearestNeighbours.h:98
 variableScope:${CMAKE_SOURCE_DIR}/Framework/API/src/FileProperty.cpp:232
 useStlAlgorithm:${CMAKE_SOURCE_DIR}/Framework/API/src/FileProperty.cpp:296
-constVariableReference:${CMAKE_SOURCE_DIR}/Framework/API/src/FileFinder.cpp:719
+constVariableReference:${CMAKE_SOURCE_DIR}/Framework/API/src/FileFinder.cpp:712
```

Comment:
While we're changing that code is there any chance we could fix this instead of moving the suppression?